README.md

```diff
@@ -1 +1,42 @@
-# nanoexp-digitalbank
\ No newline at end of file
+# nanoexp-digitalbank
+
+* Project name:  nanoexp-digitalbank
+* Author: Lucía González
+* Date:  17/12/2021
+* Version: 1.0
+
+
+## Brief introduccion 
+This project presents an accurate selection of tests applied to the Digital Bank website.
+This readme file mentions the requirements and establishes a guide for running those tests.
+Getting Started:
+
+
+## Prerequisites
+1. Install Java SDK
+2. Install node.js
+3. Node Package Manager (npm)
+
+4. Install Visual Studio Code IDE
+- wdio.config
+- npm install @wdio/cli
```

Comment:
This is not needed. This readme needs a section to explicitly indicate how to install dependencies for the project (soemthing you achive by simply running `npm install`)